.github/workflows/ci.yml

```diff
@@ -194,6 +194,12 @@ jobs:
           cat >> ~/.bash_profile << ___
             cd "$(pwd)"
 
+            # Configure canary build.
+            if [ "$DENO_CANARY" == "true" ]
```

Comment:
@piscisaureus Shouldn't this have parens?

Reply:
With parens, that would (try to) run the command DENO_CANARY right?

Reply:
Ah right. Nevermind.